server/src/main/resources/config/aws.conf

```diff
@@ -1,17 +1,13 @@
-aws {
-  credentials {
-    access_key_id = ${?AWS_ACCESS_KEY_ID}
-    secret_key = ${?AWS_SECRET_ACCESS_KEY}
-  }
-}
-
 
 s3 {
   data {
     credentials {
+      access_key_id = "some-access-key-id"
```

Comment:
I have added in yaml and build is working now . Do want to remove defaults ? May it is better to fail if there are no relevant values supplied

Reply:
Yeah I'm never sure which is the better way. We don't really want to crash the app on deployment just because a one of these values is missing, but we do want to know about it sooner rather than later so we can fix it. I wonder if one way would be to monitor for missing config values in sysdig and alert us via slack. But that's one for a future ticket